Neos.ContentRepository.Export/Tests/Behavior/Features/Import/Import.feature

```diff
@@ -0,0 +1,87 @@
+@contentrepository
+Feature: As a user of the CR I want to export the event stream
+  Background:
+    Given using no content dimensions
+    And using the following node types:
+    """yaml
+    Vendor.Site:HomePage':
+      superTypes:
+        Neos.Neos:Site: true
+    """
+    And using identifier "default", I define a content repository
+    And I am in content repository "default"
+
+  Scenario: Import the event stream
```

Comment:
Just a nitpick: Shouldn't the scenario description from line 14 be switched with line 37? Especially as the identifier is explicitly set here and not the one set in the events.json.